installing/installing_oci/installing-oci-agent-based-installer.adoc

```diff
@@ -39,6 +57,7 @@ include::modules/creating-config-files-cluster-install-oci.adoc[leveloffset=+1]
 * xref:../../installing/overview/installing-preparing.adoc#installing-preparing-selecting-cluster-type[Selecting a cluster installation type]
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#about-the-agent-based-installer[Preparing to install with the Agent-based Installer]
 * xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Downloading the Agent-based Installer]
+* xref:../../disconnected/mirroring/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red{nbsp}Hat OpenShift]
```

Comment:
```suggestion
* xref:../../disconnected/mirroring/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[Creating a mirror registry with {product-mirror-registry}]
```
Optional - this would remove the "OpenShift" in plain text but that attribute seems to be capitalized incorrectly. Pick your poison :sweat_smile: